src/rendergraph/opengl/backend/shadercache.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <map>
+
+#include "rendergraph/material.h"
+#include "rendergraph/materialshader.h"
+#include "rendergraph/materialtype.h"
+
+namespace rendergraph {
+class ShaderCache;
+}
+
+class rendergraph::ShaderCache {
+  private:
+    static std::map<MaterialType*,
+            std::shared_ptr<MaterialShader>>&
```

Comment:
why map instead of for example `std::unordered_map`?

Reply:
Yes, unordered map will suffice.